wagtail/contrib/redirects/management/commands/import_redirects.py

```diff
@@ -0,0 +1,182 @@
+import os
+
+import tablib
+from django.core.management.base import BaseCommand
+from tablib.formats import registry
+from wagtail.contrib.redirects.forms import RedirectForm
+from wagtail.core.models import Site
+
+
+class Command(BaseCommand):
+    help = "Imports redirects from .csv, .xls, .xlsx"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--src", help="Path to file", type=str, required=True,
+        )
+        parser.add_argument(
+            "--site_id", help="The site where redirects will be associated", type=int,
+        )
+        parser.add_argument(
+            "--permanent",
+            help="Save redirects as permanent redirects",
+            type=bool,
+            default=True,
+        )
+        parser.add_argument(
+            "--from_index",
+            help="The column where to read from link",
+            default=0,
+            type=int,
+        )
+        parser.add_argument(
+            "--to_index", help="The column where to read to link", default=1, type=int,
+        )
+        parser.add_argument(
+            "--dry_run",
```

Comment:
Not sure on the best way forward here, but it appears that Django commands all use `kebab-case` for their commands, so `--dry-run` instead of `--dry_run`. However, after digging into other Wagtail commands, they appear to use `underscore_case`.

This might cause confusion for Django users, as `--dry-run` seems a pretty standard usage as kebab case across other commands. 

e.g. https://docs.djangoproject.com/en/3.0/ref/django-admin/#cmdoption-makemigrations-dry-run

Reply:
In that case I'll add support for both kebab and snake to keep everyone happy (I'm personally not a `kebab-fan` casing fan since it makes copying harder)..

Reply:
I actually agree with that, double clicking on the text to copy :) 
However, alas, dealing with legacy conventions across Django and Wagtail means compromises here and there.

Thanks